src/processes/follow-topic-process.js

```diff
@@ -10,6 +10,7 @@ const {
 const BetterSocialConstantListUtils = require('../utils/constantList/utils');
 const {CHANNEL_TYPE_ANONYMOUS} = require('../utils/constant');
 const updateBetterChannelMembers = require('../getstream/updateBetterChannelMembers');
+const sendChatNotificationByChannelMembers = require('../services/fcmToken/sendChatNotificationByChannelMembers');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new import `sendChatNotificationByChannelMembers` is added correctly. It seems to be used later in the code for sending notifications.
```diff
+ const sendChatNotificationByChannelMembers = require('../services/fcmToken/sendChatNotificationByChannelMembers');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->